hoomd/md/constrain.py

```diff
@@ -144,10 +126,10 @@ class Rigid(Constraint):
     .. rubric:: Constituent particles
 
     Select one or more particle types in the simulation to use as the central
-    particles. For each rigid body particle type, ou specify the constituent
-    particle position and orientations in body coordinates (see `body`). Then,
-    `Rigid` takes control of the constituent particles and sets their position
-    and orientation in the simulation box relative to the position and
+    particles. For each rigid body particle type, you set the constituent
```

Comment:
```suggestion
    particles. For each rigid body particle type, set the constituent
```
Try to stay away from using the word "you" in documentation if possible.

Reply:
OK, I did so in `Rigid`. Fixing the rest of the documentation is far beyond the scope intended for this pull request.